packages/harpguru-core/src/components/notification-flash/notification-flash.tsx

```diff
@@ -63,8 +63,8 @@ export const NotificationFlash = ({
           {
             transform: [
               {
-                translateX: translateX,
```

Comment:
It'll be worth adding a comment here to identify that the order is important for iOS and to beware.

Reply:
Done